composer.json

```diff
@@ -7,7 +7,13 @@
         "php": ">=5.4"
     },
     "php-ext": {
+        "extension-name": "timezonedb",
         "priority": 20,
-        "configure-options": []
+        "configure-options": [
+            {
+                "name": "enable-timezonedb",
```

Comment:
This `--enable-timezonedb` option is defined in `config.m4` so it should be possible to provide this in the `configure-options` when using PIE

Reply:
But it is pointless. Using it is the default, doing ``--disable-timezonedb`` makes the extension not work.